frontend/ui/subview.jsx

```diff
@@ -170,42 +191,88 @@ export class SubViewComponent extends React.Component {
                   </tr>
                 </thead>
                 <tbody>
-                  {this.props.measures
-                    .sort((m, n) => m.name > n.name)
-                    .map(measure => (
-                      <tr key={measure.name}>
-                        <td>
-                          <a
-                            href={`#/${this.state.channel}/${
-                              this.state.platform
-                            }/${measure.name}`}>
-                            {measure.name}
-                          </a>
-                        </td>
-                        {this.props.versions.map(versionStr => (
-                          <td
-                            key={`${measure.name}-${versionStr}`}
-                            title={
-                              versionStr.includes('.')
-                                ? `Events when ${versionStr} was latest`
-                                : `All values from all point releases during selected time window when ${versionStr} was latest`
-                            }>
-                            {getReleaseValue(
-                              measure.versions.find(
-                                release => release.version === versionStr
-                              ),
-                              this.state.countType,
-                              this.state.timeWindow
-                            )}
-                          </td>
-                        ))}
-                        <td al>
-                          {measure.lastUpdated
-                            ? moment(measure.lastUpdated).fromNow()
-                            : 'N/A'}
+                  {this.state.measures.main.map(measure => (
```

Comment:
We should probably factor this into a react component like `MeasureTable` which takes a `measures` argument as input, to reduce duplication and make it easier to navigate the code. See the `PlatformCard.jsx` file for an example of this sort of thing.

Reply:
Updated. :+1: